doc/dune-files.rst

```diff
@@ -1637,7 +1652,7 @@ which for each ``g_mod`` in ``<mlg_list>`` is equivalent to:
 
 .. _dune-workspace:
 
-mdx (since 2.4)
+mdx (since 2.5)
```

Comment:
Is this change deliberate? I'm not up to speed on mdx development, but I can see the changelog for 2.4 mentioning mdx.

Reply:
Remember that check Andrey added to make sure that the manual stays up to date and always uses the latest version of the dune language in examples? In this PR, I bump the version of the dune language, hence the examples must be updated.

Reply:
I'm still confused. Isn't this an example of what *shouldn't* be updated? If mdx was introduced in 2.4 it should say "since 2.4", no?

Reply:
Oh. yes indeed. I will revert this.